docs/quick-visual-tutorials/visual-ibac-istio-authorization-policies.mdx

```diff
@@ -8,15 +8,322 @@ import CodeBlock from "@theme/CodeBlock";
+
+Then run the Helm commands shown there, and specifically follow the instructions to install Otterize <b>with enforcement on</b> (not in shadow mode). Namely, <b>omit</b> the following flag in the Helm command:
+
+`--set intentsOperator.operator.enableEnforcement=false`
+
+Let's look at the access graph again:
+
+![Access graph](/img/quick-tutorials/shadow-mode/phase-5.png)
+
+Note that all (but two) of the lock icons are locked, indicating the services are protected. And all the locks and edges are green, indicating no call attempts (discovered by the network mapper) are being blocked.
+
+:::tip How would blocked access attempts look now?
+From now on, if a client attempts a server call that wasn't covered by one of the declared intents, that would be discovered by the network mapper and show up as (new) discovered intents. Remember that the network mapper discovers attempted access, not just successful access. In this case, a red line would appear from that client to that server, and the lock on that server would turn red: calls from that client are being blocked.
+
+That may be because:
+- The calls didn't happen when the network mapper was building its map from which the intents were bootstrapped, in which case you may choose to generate all the intents again, or or just create and apply the new ones manually.
+- Or... the client maliciously called this server, but is being blocked by the authorization policies. IBAC has saved the day!
+:::
+
+<details>
+<summary>Optional: see the generated authorization policies</summary>
+Otterize automatically generated authorization policies according to your declared intents.
+
+To list all generated authorization policies run:
+
+```bash
+kubectl get authorizationpolicies.security.istio.io -n otterize-visual-tutorial-istio
+```
+
+Let's inspect one of these authorization policies with:
+```bash
+kubectl get authorizationpolicies.security.istio.io -n otterize-visual-tutorial-istio authorization-policy-to-productcatalogservice-from-checkoutservice.otterize-visual-tutorial-istio -o yaml
+```
+
+The result should be:
+
+```yaml
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: authorization-policy-to-productcatalogservice-from-checkoutservice.otterize-visual-tutorial-istio
+  namespace: otterize-visual-tutorial-istio
+...
+spec:
+  rules:
+  - from:
+    - source:
+      principals:
+      - cluster.local/ns/otterize-visual-tutorial-istio/sa/checkoutservice-service-account
+  to:
+    - operation:
+      methods:
+      - POST
+      paths:
+      - /products
+  selector:
+    matchLabels:
+      intents.otterize.com/server: productcatalogservic-otterize-visual-tuto-99a036
+```
+</details>
+
+## What's next
+
+- Learn how to [manage secure access for Kafka](/quick-visual-tutorials/visual-ibac-kafka-k8s) using the demo lab tutorial.
+
```

Comment:
What should be our next step? Netpol, Kafka? I feel like it should be our "best practices" or "Pushing IBAC to your production CI/CD" page

Reply:
@tomergreenwald reminder about creating a doc for roll out scenarios.